app/hooks/useLiquidityPosition.ts

```diff
@@ -67,10 +67,12 @@ export function useLiquidityPosition({
   const wallet = walletsState![getChainKeyFromChain(assetChain)];
   // Determine if this is a UTXO chain and which one
   const utxoChain = useMemo(() => {
-    const chain = assetChain.toLowerCase();
-    if (chain === "btc") return "BTC";
-    if (chain === "doge") return "DOGE";
-    return null;
+    const chainMap: Record<string, string> = {
```

Comment:
Non-blocking: this should be part of chain config, but I don't think it's necessary at all. assetChain be a type that satisfies the ChainConfig interface imo